src/components/Callout/CalloutContent.tsx

```diff
@@ -134,9 +134,10 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
     if (hostElement && calloutElement) {
       let positionInfo: IPositionInfo = getRelativePositions(this.props, hostElement, calloutElement);
 
-      // Set the new position only when the positions are not exists or one of the new callout positions are different
+      // Set the new position only when the positions are not exists or one of the new callout positions are different.
+      // The positon should not change if the position is within 2 decimal places.
       if ((!positions && positionInfo) ||
-        (positions && positionInfo && (positions.callout.top !== positionInfo.calloutPosition.top || positions.callout.left !== positionInfo.calloutPosition.left))) {
+        (positions && positionInfo && (positions.callout.top.toFixed(2) !== positionInfo.calloutPosition.top.toFixed(2) || positions.callout.left.toFixed(2) !== positionInfo.calloutPosition.left.toFixed(2)))) {
```

Comment:
You are correct, 2 decimal places doesn't matter, 1 decimal place, however, does. Check out this codepen. http://codepen.io/slimemold/pen/amVJPp?editors=1111 . You can see that a decimal pixel point is automatically rounded up if the decimal is .5 or greater. It will also round up if you have 8 or more 9s. So 1.49999999px = 2px but that is an unlikely scenario.